src/main/java/it/pn/frontend/e2e/pages/destinatario/personaGiuridica/DeleghePGPagoPAPage.java

```diff
@@ -332,6 +332,56 @@ public boolean siVisualizzaUnaDelegaPG() {
         }
     }
 
+    public void checkAlertWrongDelegationCode() {
+        try {
+            By alertWrongCode = By.id("alert-api-status}");
+            By alertCloseButtonBy = By.xpath("//*[@aria-label='Close']");
```

Comment:
Specify the correct element (instead of `*`) you want to select to prevent any potential issues in the future